### PR TITLE
Improves Repo's Code of Conducts, Contributing, and How-to Indonesian Grammars (review assistance requested)

### PR DESCRIPTION
**Pull request by NikarashiHatsu**
## What does this PR do?
Improve repo

### Description
This PR improves the repo's Indonesia translations of Code of Conducts, Contributing, and How-to.

### Why is this valuable (or not)?
The intention of this PR is to make the Indonesia translation files looks more-friendly and less intimidating.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
need review help for indonesian

File: docs/CONTRIBUTING-id.md [hunk 1]
```diff
@@ -1,68 +1,67 @@
...
-- Kami tidak merekam file yang ada di Google Drive, Dropbox, Mega, Scribd, Issuu atau platform unggah file lainnya sebanding.
-- Masukkan tautan dalam urutan abjad, as described [below](#alphabetical-order).
-- Selalu pilih tautan dari sumber otoritatif (yaitu, situs web penulis lebih baik daripada situs web editor, yang pada gilirannya akan lebih baik daripada situs web pihak ketiga)
-    - tidak ada platform hosting file (termasuk tautan ke Dropbox, Google Drive, dll.).
-- Tautan `https` harus selalu lebih disukai daripada tautan `http` - selama tautan tersebut mengarah ke domain dan konten yang sama.
-- Garis miring harus dihapus pada domain root: `http://example.com` alih-alih `http://example.com/`
+- Pastikan bahwa buku yang Anda tambahkan benar-benar gratis. Periksa dua kali jika perlu. Ini akan membantu para admin jika Anda memberikan komentar di PR mengenai alasan Anda menganggap buku tersebut gratis.
+- Kami tidak menerima file yang dihosting di Google Drive, Dropbox, Mega, Scribd, Issuu, dan platform unggah file serupa lainnya.
```
Comment (muhfaris) [inline docs/CONTRIBUTING-id.md:42]: I though the word 'dihosting' should be changed to 'diunggah'.

Comment (NikarashiHatsu) [reply docs/CONTRIBUTING-id.md:42]: Have you read the original [CONTRIBUTING.md](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md) docs?

It simply says `hosted`, not `uploaded` which means to `diunggah`.

So I guess there's no change needed.

Comment (muhfaris) [reply docs/CONTRIBUTING-id.md:42]: just curious, why not to be better translated? 
I think this guide for indonesian people, used the good sentences is better instead terms.

I guess it's can be translated like
- Kami tidak menerima file dari Google Drive, ...
- Kami tidak menerima file yang bersumber dari Google Drive, ...

Comment (NikarashiHatsu) [reply docs/CONTRIBUTING-id.md:42]: Hey, that's actually even nicer. Didn't think of that.
Let me change it that way.

Comment (NikarashiHatsu) [reply docs/CONTRIBUTING-id.md]: Thank you for your suggestion, that definitely looks better.

File: docs/CONTRIBUTING-id.md [hunk 2]
```diff
@@ -1,68 +1,67 @@
...
+    - *Kumpulan Masalah & Pemrograman Kompetitif*: Situs web atau perangkat lunak yang memungkinkan Anda mengukur kemampuan pemrograman Anda dengan menyelesaikan masalah-masalah sederhana atau kompleks, dengan atau tanpa tinjauan kode, dengan atau tanpa membandingkan hasilnya dengan pengguna lain.
 
-4. Pastikan Anda mengikuti [Guidelines](#guidelines) dan hormati [Markdown Formatting](#formatting) dari file.
+4. Pastikan Anda mengikuti [Guidelines di bawah](#guidelines) dan menghormati [Formatting Markdown](#formatting) dari setiap file.
 
-5. GitHub Actions akan menjalankan tes untuk memastikan bahwa **daftar diurutkan berdasarkan abjad dengan benar** dan bahwa **aturan pemformatan telah diikuti**. **Pastikan** perubahan Anda lulus tes ini.
+5. GitHub Actions akan menjalankan pengujian untuk **memastikan bahwa daftar yang Anda buat diurutkan secara alfabetis** dan **mengikuti aturan format**. **Pastikan** untuk memeriksa bahwa perubahan yang Anda buat lulus uji coba tersebut.
```
[the remark this answers is not in the document]
Comment (NikarashiHatsu) [reply docs/CONTRIBUTING-id.md:35]: Sure thing, that looks neat

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books, all three of you!

**Pull request merged**